rpm/mce.spec

```diff
@@ -20,6 +20,7 @@ BuildRequires:  pkgconfig(thermalmanager_dbus_if)
 BuildRequires:  pkgconfig(libiphb)
 BuildRequires:  pkgconfig(glib-2.0) >= 2.36.0
 BuildRequires:  pkgconfig(mce) >= 1.30.0
+#                                 ^^^^^^ TBD/1.31.0
```

Comment:
TBD

Reply:
Set to:
BuildRequires:  pkgconfig(mce) >= 1.31.0